client.go

```diff
@@ -545,6 +545,21 @@ func (client *gocloak) LoginClientTokenExchange(ctx context.Context, clientID, t
 	})
 }
 
+// LoginSocialTokenExchange will exchange the presented token for a user's token
+// Requires Token-Exchange is enabled: https://www.keycloak.org/docs/latest/securing_apps/index.html#external-token-to-internal-token-exchange
+func (client *gocloak) LoginSocialTokenExchange(ctx context.Context, clientID, token, clientSecret, realm, targetClient, issuer string) (*JWT, error) {
```

Comment:
Hi, could you please add a test for that function? :)

Reply:
okhay bro @Nerzal